spec/PeriodicBackgroundSync-index.bs

```diff
@@ -157,3 +159,51 @@ The <code><dfn method for=PeriodicSyncManager title="getTags()">getTags()</dfn><
 1. Let <var>currentTags</var> be a new [=sequence=].
 1. For each <var>registration</var> in |serviceWorkerRegistration|'s [=list of periodicsync registrations=], add |registration|'s [=tag=] to |currentTags|.
 1. [=Resolve=] |promise| with |currentTags|.
+
+## The <dfn>periodicSync event</dfn> ## {#periodicSync-event}
+<script type="idl">
+dictionary PeriodicSyncEventInit : ExtendableEventInit {
+    required DOMString tag;
+};
+
+[
+    Constructor(DOMString type, PeriodicSyncEventInit init),
+    Exposed=ServiceWorker
+] interface PeriodicSyncEvent : ExtendableEvent {
+    readonly attribute DOMString tag;
+  };
+</script>
+
+The {{PeriodicSyncEvent}} interface represents a [=firing=] [=periodicsync registration=].
+
+### Firing periodicSync events ### {#firing-periodicsync-events}
+Whenever the user agent changes to [=online=], it SHOULD [=fire a periodicSync event=] for each [=periodicsync registration=] whose [=registration state=] is [=pending=] and [=time to fire=] is now or in the past. The events MAY be fired in any order.
+
+### [=Fire a periodicSync event=] ### {#firing-a-periodicsync-event}
+The user agent will fire a [=periodicSync event=] for a [=periodicsync registration=] as soon as network connectivity is available, at or after the [=time to fire=] of the [=periodicsync registration=].
+If a [=periodicSync event=] fails, the user agent MAY <dfn>retry</dfn> it one or more times at a time of its choosing, based on some user agent defined heuristics. The interval between [=retry=] attempts SHOULD be much smaller than the [=minInterval=] value associated with the [=periodicsync registration=].
```

Comment:
"much smaller" is unclear. What I think you mean to say here is that it should be tried again as soon as possible, but may be subject to a user agent-specific back-off?

Reply:
Yes, I've removed this from here since it's now covered in the "calculate a time to fire"